Tiltfile

```diff
@@ -6,10 +6,11 @@ default_registry(settings.get('default_registry'))
 
 docker_build("controller:latest", ".", dockerfile='Dockerfile.tilt',
   live_update=[
-    sync('.', '/go/src/github.com/pivotal/kubernetes-image-mapper/'),
-    run('CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o /manager main.go'),
+    sync('.', '/workspace'),
+    run('cd /workspace && go mod download', trigger=['go.mod', 'go.sum']),
```

Comment:
There's no point running this separately. I think that was done to improve the caching of layers with a different dockerfile structure.